build.gradle.kts

```diff
@@ -593,7 +596,14 @@ tasks {
   val outputDir = projectDir.dir("distribution")
 
   val compileDistribution by registering {
-    dependsOn(":web:web:build", "copySubprojectDependencies", "copyCatalogLibAndConfigs", ":authorizations:copyLibAndConfig", "copySubprojectLib", "iceberg:iceberg-rest-server:copyLibAndConfigs")
+    dependsOn(
+      "copyCatalogLibAndConfigs",
+      "copySubprojectDependencies",
+      "copySubprojectLib",
+      ":authorizations:copyLibAndConfig",
+      ":iceberg:iceberg-rest-server:copyLibAndConfigs",
```

Comment:
To reviwers: by wrapping the long line, the missing leading ':' for iceberg one becomes obvious.